pkgs/by-name/vi/vidmerger/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, ffmpeg
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vidmerger";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "TGotwig";
+    repo = "vidmerger";
+    rev = version;
+    hash = "sha256-TnVDbhMPBByuhXNuKiyhq6wD3XZgp8nGtIf+7XkVmw8=";
+  };
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    cp ${./Cargo.lock} Cargo.lock
+  '';
+
+  # Running cargo test -- . fails
+  doCheck = false;
```

Comment:
This needs more context. Why are they failing, and is it reasonable to leave them as such?